ang/workflow/directives/workflow-list.directive.js

```diff
@@ -35,6 +36,11 @@
     $scope.tableColumns = filterArrayForCurrentInstance(WorkflowListColumns);
     $scope.filters = filterArrayForCurrentInstance(WorkflowListFilters);
     $scope.selectedFilters = {};
+    $scope.currentCaseCategory = CaseTypeCategory.findByName(currentCaseCategory);
+    $scope.currentCaseCategory.singular_label = _.first(
+      $scope.currentCaseCategory['api.CaseCategoryInstance.get'].values
```

Comment:
Lets use a function to fetch `api.CaseCategoryInstance.get`, also please assign this to a meaningful "key" in the provider.
```javascript
_.map(caseTypeCategories, function (caseTypeCategory) {
  return {
    ..
    singularLabel: _.first(caseTypeCategory.['api.CaseCategoryInstance.get'].values).singular_label
  };
})